src/NServiceBus.Transport.RabbitMQ/RabbitMQTransport.cs

```diff
@@ -199,6 +199,11 @@ public TimeSpan NetworkRecoveryInterval
             }
         }
 
+        /// <summary>
+        /// Should this endpoint's incoming queues be created as quorum queues.
+        /// </summary>
+        public bool IsEndpointQuorum { get; set; } = true;
```

Comment:
I think we should use a name that better match the rabbitmq concept to make it more clear that this isn't a NServiceBus feature. How about `UseRabbitMqQuorumQueues` or `UseQuorumQueues` ?